BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -135,17 +152,20 @@ extension ElementContent {
     /// Initializes a new `ElementContent` with no children that delegates to the provided measure function.
     ///
     /// - parameter measurementCachingKey: An optional key to use to cache measurement. See the `MeasurementCachingKey` documentation for more.
-    /// - parameter measureFunction: How to measure the `ElementContent` in the given `SizeConstraint`.
+    /// - parameter measureFunction: How to measure the `ElementContent` in the given `SizeConstraint` and `LayoutContext`.
     public init(
```

Comment:
While in here; it might be nice to swap these out with static funcs that make it a bit easier to get to the init you want. Eg, I always end up searching through autocomplete for the one init I want, whereas it'd be a lot easier to find the one you want if we switched to:

```
var content : ElementContent {
    .byMeasuring { constraint, context in
        ....
    }
}

var content : ElementContent {
    .withChild { constraint, context in
        ....
    }
}
```